packages/react/src/ButtonGroup/ButtonGroup.tsx

```diff
@@ -20,6 +20,7 @@ export const ButtonGroup = forwardRef(
             size: buttonSize,
             as: buttonsAs,
             variant: index === 0 ? 'primary' : 'secondary',
+            hasArrow: index === 0 ? true : false,
```

Comment:
Possible to add a test around the default behaviour?

Reply:
Yes I will add a test specifically for this on this component and the button group